04_removeFromArray/README.md

```diff
@@ -10,7 +10,7 @@ removeFromArray([1, 2, 3, 4], 3); // should remove 3 and return [1,2,4]
 
 The first test on this one is fairly easy, but there are a few things to think about(or google) here for the later tests:
 
+- you can manipulate the original array you pass into the function call or create a new array that is returned as the result.
 - how to remove a single element from an array
 - how to deal with multiple optional arguments in a javascript function
 - [Check this link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments).  Scroll down to the bit about `Array.from` or the spread operator. - [Or this link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters).
```

Comment:
Since we're in here can we just update the link text for these two links to be more descriptive than "this"?